pa1/PPCodeUnitStream.h

```diff
@@ -0,0 +1,55 @@
+#ifndef PPCodeUnitStream_h
+#define PPCodeUnitStream_h
+
+#include "PPCodeUnit.h"
+#include "UTF32StreamIfc.h"
+#include "PPCodeUnitStreamIfc.h"
+#include <queue>
+#include <string>
+
+// Stream UTF32 code points into stream of PPCodeUnit that is easy to tokenize.
+//
+//   - Line splicing. i.e., occurrences of "\\\n" are replaced with
+//     PPCodeUnitType::WhitespaceCharacter so that we can revert it in raw strings.
+//
```

Comment:
Comments and digraphs are no longer parsed in the PPCodeUnitStream class. Should update the comment to reflect the changes. Might be good to shed light upon why they cannot be treated as PPCodeUnits: potential collision with raw string d-char.